documentation/modules/auxiliary/scanner/mssql/mssql_login.md

```diff
@@ -0,0 +1,43 @@
+## Vulnerable Application
+
+This module simply queries the MSSQL instance for a specific user/pass (default is sa with blank). 
+
+## Verification Steps
+Example steps in this format (is also in the PR):
+
+1. Start msfconsole
+2. Do: ```use scanner/mssql/mssql_login```
+3. Do: ```set RHOSTS [IP]```
+4. Do: ```run```
+5. You should get a shell.
+
+## Options
+A number of options interesting options without default values exist. They are as follows: 
+   
+   Name                 Current Setting  Required  Description
+   ----                 ---------------  --------  -----------
+   PASSWORD                              no        A specific password to authenticate with
+   PASS_FILE                             no        File containing passwords, one per line
+   RHOSTS                                yes       The target host(s),
+   USERPASS_FILE                         no        File containing users and passwords separated by space, one pair per line
+   USER_FILE                             no        File containing usernames, one per line
+
+Options ending in _FILE are used by specifing a file location. For example, specifying the USER_FILE would happend as follows:
+
+```
+msf6 auxiliary(scanner/mssql/mssql_login) > set USER_FILE ./userfile
+```
+
+
+## Scenarios
+Specific demo of using the module that might be useful in a real world scenario.
+```
+msf > use scanner/mssql/mssql_login
+msf (auxiliary(scanner/mssql/mssql_login)) > set RHOSTS 178.33.113.209
+msf (auxiliary(scanner/mssql/mssql_login)) > run 
+
+[*] 59.36.92.188:1433     - 59.36.92.189:1433 - MSSQL - Starting authentication scanner.
+[-] 59.36.92.188:1433     - 59.36.92.189:1433 - LOGIN FAILED: WORKSTATION\sa: (Unable to Connect: The connection with (59.36.92.188:1433) timed out.)
+[*] 59.36.92.188:1433     - Scanned 1 of 1 hosts (100% complete)
+[*] Auxiliary module execution completed
```

Comment:
We usually reserve the `Scenarios` section for demonstrations of successful module runs, so that people trying to use the module in the future have a reference for how to get it working. 
```suggestion
msf > use scanner/mssql/mssql_login
msf6 auxiliary(scanner/mssql/mssql_login) > set rhosts 127.0.0.1
rhosts => 127.0.0.1
msf6 auxiliary(scanner/mssql/mssql_login) > set password N0tpassword!
password => N0tpassword!
msf6 auxiliary(scanner/mssql/mssql_login) > options

Module options (auxiliary/scanner/mssql/mssql_login):

   Name                 Current Setting  Required  Description
   ----                 ---------------  --------  -----------
   ANONYMOUS_LOGIN      false            yes       Attempt to login with a blank username and password
   BLANK_PASSWORDS      true             no        Try blank passwords for all users
   BRUTEFORCE_SPEED     5                yes       How fast to bruteforce, from 0 to 5
   DB_ALL_CREDS         false            no        Try each user/password couple stored in the current database
   DB_ALL_PASS          false            no        Add all passwords in the current database to the list
   DB_ALL_USERS         false            no        Add all users in the current database to the list
   DB_SKIP_EXISTING     none             no        Skip existing credentials stored in the current database (Accepted: none, user, user&realm)
   PASSWORD             N0tpassword!     no        A specific password to authenticate with
   PASS_FILE                             no        File containing passwords, one per line
   RHOSTS               127.0.0.1        yes       The target host(s), see https://docs.metasploit.com/docs/using-metasploit/basics/using-metasploit.html
   RPORT                1433             yes       The target port (TCP)
   STOP_ON_SUCCESS      false            yes       Stop guessing when a credential works for a host
   TDSENCRYPTION        false            yes       Use TLS/SSL for TDS data "Force Encryption"
   THREADS              1                yes       The number of concurrent threads (max one per host)
   USERNAME             sa               no        A specific username to authenticate as
   USERPASS_FILE                         no        File containing users and passwords separated by space, one pair per line
   USER_AS_PASS         false            no        Try the username as the password for all users
   USER_FILE                             no        File containing usernames, one per line
   USE_WINDOWS_AUTHENT  false            yes       Use windows authentication (requires DOMAIN option set)
   VERBOSE              true             yes       Whether to print output for all attempts


View the full module info with the info, or info -d command.

msf6 auxiliary(scanner/mssql/mssql_login) > run

[*] 127.0.0.1:1433        - 127.0.0.1:1433 - MSSQL - Starting authentication scanner.
[!] 127.0.0.1:1433        - No active DB -- Credential data will not be saved!
[+] 127.0.0.1:1433        - 127.0.0.1:1433 - Login Successful: WORKSTATION\sa:N0tpassword!
[*] 127.0.0.1:1433        - Scanned 1 of 1 hosts (100% complete)
[*] Auxiliary module execution completed
msf6 auxiliary(scanner/mssql/mssql_login) >
```